packages/shared/src/libs/types/token-payload.type.ts

```diff
@@ -1,4 +1,5 @@
 type TokenPayload = {
+	email?: string;
```

Comment:
might be trash since i don't send email inside of jwtToken used to create the link